volume/volume.go

```diff
@@ -84,6 +84,18 @@ type VolumeDriver interface {
 	Enumerator
 }
 
+// VolumeClient contains the exported functions to external clients.
+type VolumeClient interface {
+	StatsDriver
+	BlockDriver
+	Enumerator
+	SnapshotDriver
+	QuiesceDriver
```

Comment:
In an offline discussion with @jvinod , he mentioned we can remove the quiesce calls from the API.

Reply:
Sweet!!

Reply:
I will do it on the next patch. I created #402